json/config.json

```diff
@@ -6,5 +6,7 @@
   "blockchainNodeUri": "http://127.0.0.1:3009",
   "batchIntervalInSeconds": 10,
   "maxConcurrentCasDownloads": 20,
-  "pollingIntervalInSeconds": 10
+  "pollingIntervalInSeconds": 10,
+  "operationStoreType": "InMemory",
```

Comment:
Sanity: `operationStoreType` should be removed?

Reply:
Duplicate? Yes. removed.